docs/deployed-contracts/sepolia.md

```diff
@@ -100,6 +100,21 @@ and permission-based [validator set](https://github.com/eth-clients/sepolia/issu
   - L2LidoGateway
   - L2WstETHToken
 
+### Mode
+
+##### Ethereum part
+
+- L1ERC20TokenBridge: [`0xD847780c8808fc99727001C43E6C17A198b39729`](https://sepolia.etherscan.io/address/0xD847780c8808fc99727001C43E6C17A198b39729) (proxy)
```

Comment:
should be [0x16B929D35B200EA0ae0B93EABc3Bf9Ad611BF18F](https://sepolia.etherscan.io/address/0x16B929D35B200EA0ae0B93EABc3Bf9Ad611BF18F#readContract)